src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Tab/_inventory.html.twig

```diff
@@ -2,7 +2,10 @@
     <h3 class="ui dividing header">{{ 'sylius.ui.inventory'|trans }}</h3>
     <div class="ui segment">
         {{ form_row(form.onHand) }}
+
         {{ form_row(form.tracked) }}
+        <div style="color: rgba(0, 0, 0, .6)">{{ form_help(form.tracked) }}</div>
```

Comment:
can we remove inline styles here as well?

Reply:
Yes.